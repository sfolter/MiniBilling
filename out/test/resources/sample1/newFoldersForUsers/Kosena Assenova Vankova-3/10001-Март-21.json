{
    "documentDate": "2022-08-08T13:39:43.929886500",
    "documentNumber": "10001",
    "consumer": "Kosena Assenova Vankova",
    "reference": "3",
    "totalAmount": 784.8,
    "lines": [{
        "index": 1,
        "quantity": 436,
        "lineStart": "2021-01-01T14:40:00Z",
        "lineEnd": "2021-03-11T07:00:00Z",
        "product": "gas",
        "price": 1.8,
        "priceList": 1,
        "amount": 784.8
    }]
}