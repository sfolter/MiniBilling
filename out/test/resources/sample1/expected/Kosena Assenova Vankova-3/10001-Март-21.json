{
  "documentNumber": "10002",
  "consumer": "Kosena Assenova Vankova",
  "reference": "3",
  "totalAmount": 784.8,
  "lines": [
    {
      "index": 1,
      "quantity": 436,
      "lineStart": "2021-01-01T14:40:00Z",
      "lineEnd": "2021-03-11T07:00:00Z",
      "product": "gas",
      "price": 1.8,
      "priceList": 1,
      "amount": 784.8
    }
  ]
}